c_emulator/riscv_sim.c

```diff
@@ -53,6 +53,7 @@ const char *RV32ISA = "RV32IMAC";
 #define OPT_ENABLE_WRITABLE_FIOM 1001
 #define OPT_PMP_COUNT 1002
 #define OPT_PMP_GRAIN 1003
+#define OPT_ENABLE_SVINVAL 10017
```

Comment:
Any reason to pick 17 here? Not that it matters, but it is easier to just keep incrementing by one? I missed that with the ZCB since it looked like 3 -> 4 but was actually 3 -> 14.

Reply:
I don't think they need to be consecutive considering how long it takes to merge even trivial PRs to this repository, but we should keep them in increasing order to see which ones are available.

Reply:
One could make this an enum with the first value assigned to 1000 so that each subsequent entry is automatically one more than the previous

Reply:
(Not for this PR, but as future cleanup; or one could not care given it's mostly going to be obsoleted by a proper -march parser)

Reply:
Agreed that there is no need to spend much time on this since it will be removed eventually. I like the enum idea though since it prevents mistakes and mismerges.

Reply:
> Any reason to pick 17 here? Not that it matters, but it is easier to just keep incrementing by one? I missed that with the ZCB since it looked like 3 -> 4 but was actually 3 -> 14.

I pulled 100017 out of thin air, because I could not see any logic in the existing numbers.  I agree that an enum would be better.

Reply:
Changed 10017 to 1004